### PR TITLE
Update Terraform category page and get started

### PR DESCRIPTION
**Pull request by staceysalamon-aiven**
# What changed, and why it matters
* Remove links to pages that are no longer being used in the new structure
* Add link to the Cookbook, which is now hosted on the Dev Center
* Reword get started and align with style guide
* Minor changes to simplify get started sample

### CODE REVIEW

**Review by Serpentiel — APPROVED** [review]
lgtm, @byashimov check this too, please

**Comment by staceysalamon-aiven** [comment]
@byashimov Whenever you have a free minute, would you mind checking this looks okay to you? :) Thanks! 🙏

File: docs/tools/terraform/get-started.rst [hunk 1]
```diff
@@ -88,64 +87,67 @@ The ``variables.tf`` file defines both the API token, and the project name to us
...
+   terraform apply
+
+The output will be similar to the following:
+
+.. code:: bash
+  
+  Apply complete! Resources: 1 added, 0 changed, 0 destroyed.
 
-The output will show you if everything worked well. You can now visit the `Aiven web console <https://console.aiven.io>`_ and admire your new services.
+You can also see the service in the `Aiven Console <https://console.aiven.io>`_.
```
Comment (byashimov) [inline docs/tools/terraform/get-started.rst:127]: nit: Terraform will ask you to type "yes". Running `apply` without `--auto-approve` flag outputs confirmation:

```
Do you want to perform these actions?
  Terraform will perform the actions described above.
  Only 'yes' will be accepted to approve.

  Enter a value: yes <- you type this "yes"
```

File: docs/tools.rst [hunk 2]
```diff
@@ -58,9 +58,9 @@ You can use the Aiven platform in the way that best fits your workflow with our
         :shadow: md
         :margin: 2 2 0 0
 
-        **Aiven Terraform Provider** 
+        **Aiven Provider for Terraform**
```
Comment (byashimov) [inline docs/tools.rst:61]: nit: I think the original wording is what people google

File: docs/tools/terraform/get-started.rst [hunk 3]
```diff
@@ -1,57 +1,56 @@
...
+This example shows you how to use the Aiven Provider to set up your data infrastructure by creating a single Aiven for Redis®* service in an :doc:`Aiven project </docs/platform/concepts/projects_accounts_access>`. 
 
-Prepare the dependencies 
-''''''''''''''''''''''''
-- `Download and install Terraform <https://www.terraform.io/downloads>`_
-- `Sign up <https://console.aiven.io/signup?utm_source=github&utm_medium=organic&utm_campaign=devportal&utm_content=repo>`_ for Aiven if you haven't already
-- `Generate an authentication token <https://docs.aiven.io/docs/platform/howto/create_authentication_token.html>`_
+.. caution::
+
+  Recreating stateful services with Terraform may delete the service and all its data before creating it again. Some properties, like project and resource name, cannot be changed and it will trigger a resource replacement. Check the Terraform plan to find out whether a service will be deleted or replaced.
```
Comment (byashimov) [inline docs/tools/terraform/get-started.rst:8]: nit: terraform plan _command_? I mean this absolutely correct sentence, but late it explains for the very first time what plan is. And the word is confusing, like if terraform usage has own "price plan". Adding a link to the below example might also help the understanding what plan is.

File: docs/tools/terraform.rst [hunk 4]
```diff
@@ -1,44 +1,39 @@
 Aiven Provider for Terraform
 =============================
 
-With Aiven's `Terraform <https://www.terraform.io>`_ provider, you can use an open-source infrastructure as code software tool to declare and manage your cloud services.
+With Aiven's `Terraform <https://www.terraform.io>`_ provider, you can use an open source infrastructure as code tool to declare and manage your cloud services.
 
-See the `Aiven Terraform provider documentation <https://registry.terraform.io/providers/aiven/aiven/latest/docs>`_ to learn about the services and resources,and visit the `GitHub repository <https://github.com/aiven/terraform-provider-aiven>`_ to report any issues or contribute to the project.
+Start using Aiven Provider
```
Comment (byashimov) [inline docs/tools/terraform.rst:6]: btw, referencing to the naming change, there will be Aiven Crossplane Provider, which is also _a provider_.

File: docs/tools.rst (hunk 2 above)
Comment (Serpentiel) [reply docs/tools.rst:61]: this is the official product name

File: docs/tools/terraform.rst (hunk 4 above)
Comment (Serpentiel) [reply docs/tools/terraform.rst:6]: Hashicorp suggests to stop using the word `Provider` and use `Plugin` instead, in every case, for the context of Terraform providers

cc @staceysalamon-aiven

File: docs/tools/terraform/get-started.rst [hunk 5]
```diff
@@ -88,64 +87,67 @@ The ``variables.tf`` file defines both the API token, and the project name to us
...
-
-The ``init`` command performs several different initialization steps in order to prepare the current working directory for use with Terraform. In our case, this command automatically finds, downloads, and installs the necessary Aiven Terraform provider plugins.
+1. The ``init`` command prepares the working directly for use with Terraform. Run this command to automatically find, download, and install the necessary Aiven Provider plugins:
 
 .. code:: bash
 
    terraform init 
 
-The ``plan`` command creates an execution plan and shows you the resources that will be created (or modified) for you. This command does not actually create any resource; this is more like a preview.
+2. Run the ``plan`` command to create an execution plan and view the resources that will be created or modified. This command does not create any resources.
```
[the remark this answers is not in the document]
Comment (staceysalamon-aiven) [reply docs/tools/terraform/get-started.rst:109]: Good point! I was focusing on the actions in this example, but this is an opportunity to educate as well. Would "Run the plan command to _preview_ the changes to the infrastructure." be clear do you think?

File: docs/tools/terraform.rst (hunk 4 above)
Comment (staceysalamon-aiven) [reply docs/tools/terraform.rst:6]: As this is used throughout Aiven's channels (website, etc.), I think any change like that would have to be part of a bigger discussion/project. For now, I've been trying to standardize with the Aiven Provider for Terraform name.

**Pull request merged**